Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/package.json

```diff
@@ -4,10 +4,15 @@
   "description": "Automated regression tests for the DfE Prepare conversions journeys",
   "main": "index.js",
   "scripts": {
-    "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "lint": "eslint . --ext .cy.js,.js,.jsx,.ts,.tsx --ignore-path .gitignore",
-    "cy:notify": "cypress-slack-reporter"
+    "cy:open": "cypress open --browser edge",
+    "cy:run": "npm run clean:reports && cypress run --browser edge",
+    "cy:zap": "npm run clean:reports && cypress run --browser electron",
+    "cy:notify": "cypress-slack-reporter",
+    "clean:reports": "rm -rf cypress/reports/mocha/*.json",
+    "combine:reports": "mochawesome-merge  cypress/reports/mocha/*.json> mochareports/report.json",
+    "create:html:report": "marge  mochareports/report.json -f report -o mochareports",
```

Comment:
Should this be merge?

Reply:
Hi Chris,

It is actually marge, which is hilarious :D

Reply:
its actually marge-json :P but it is what it is :P

Reply:
You learn something new every day!